### ISSUE TITLE
socket hang up Fedora 32

### ISSUE THREAD

**Issue by TheGP** [mattcg/socks5-client#21]
Fedora 32 socket5 proxy emits: (but on Ubuntu/Mac it works well) Node v15.9.0

`socket hang up
    at connResetException (node:internal/errors:642:14)
    at Socks5ClientSocket.socketOnEnd (node:_http_client:486:23)
    at Socks5ClientSocket.emit (node:events:390:22)
    at TLSSocket.<anonymous> (/root/dotalaning/node_modules/socks5-client/lib/Socket.js:146:8)
    at TLSSocket.emit (node:events:390:22)
    at endReadableNT (node:internal/streams/readable:1307:12)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {
  code: 'ECONNRESET'
} {
`

**Comment by aicanfly** [comment]
did you fix it somehow?

**Comment by TheGP** [comment]
Nope, I switched to `curlrequest` library and making requests with curl which supports socket5 proxies and many others.
